irs/adapters/inbound/s3_download.py

```diff
@@ -49,16 +49,20 @@ def calc_part_ranges(
     return part_ranges
 
 
-async def get_download_url(*, object_id: str) -> str:
-    """Get object download URL from S3 inbox bucket"""
+def get_objectstorage() -> S3ObjectStorage:
+    """Factoring this out makes it overridable by tests"""
     config = S3Config(
         s3_endpoint_url=CONFIG.s3_endpoint_url,
         s3_access_key_id=CONFIG.s3_access_key_id,
         s3_secret_access_key=CONFIG.s3_secret_access_key,
     )
-    storage = S3ObjectStorage(config=config)
+    return S3ObjectStorage(config=config)
```

Comment:
PMO: this can be achieved using hexkit's dependency injection solution.
But it's fine like this for now.